### PR TITLE
fix flytekit docker image name 

### PR DESCRIPTION
**Pull request by evalsocket**
# TL;DR
1. Fix the flytekit docker image name 

Tested Action Result : https://github.com/users/evalsocket/packages/container/package/flytekit

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

File: .github/workflows/pythonpublish.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ jobs:
           # https://docs.github.com/en/packages/learn-github-packages/publishing-a-package
           username: "${{ secrets.FLYTE_BOT_USERNAME }}"
           password: "${{ secrets.FLYTE_BOT_PAT }}"
-          image_name: ${{ github.repository_owner }}/flytekit
+          image_name: ${{ github.repository_owner }}/flytekit-py37
```
Comment (EngHabu) [inline .github/workflows/pythonpublish.yml:61]: Can we keep the name of the image the same but add this as a prefix to the version?

File: .github/workflows/pythonpublish.yml [hunk 2]
```diff
@@ -58,7 +58,7 @@ jobs:
...
           image_tag: flytekit-latest,flytekit-${{ github.sha }},flytekit-${{ steps.bump.outputs.version }}
```
Comment (EngHabu) [inline .github/workflows/pythonpublish.yml:62]: Something like this:
```suggestion
          image_tag: py37-latest,py37-${{ github.sha }},py37-${{ steps.bump.outputs.version }}
```

Comment (evalsocket) [reply .github/workflows/pythonpublish.yml]: ok

File: .github/workflows/pythonpublish.yml (hunk 1 above)
Comment (evalsocket) [reply .github/workflows/pythonpublish.yml]: yes that's a good idea

**Review by wild-endeavor — APPROVED** [review]
